app/screens/image_preview/image_preview.js

```diff
@@ -197,6 +208,28 @@ export default class ImagePreview extends PureComponent {
         }
     };
 
+    showDeletedFilesAlert = () => {
+        const {intl} = this.context;
+
+        Alert.alert(
+            intl.formatMessage({
+                id: 'mobile.image_preview.deleted_post_title',
+                defaultMessage: 'Post Deleted'
+            }),
+            intl.formatMessage({
+                id: 'mobile.image_preview.deleted_post_message',
+                defaultMessage: "This post and it's files have been deleted. The previewer will now be closed."
+            }),
+            [{
+                text: intl.formatMessage({
+                    id: 'mobile.server_upgrade.button',
+                    defaultMessage: 'OK'
+                }),
+                onPress: () => this.close()
```

Comment:
This can just be `onPress: this.close` without the arrow function. Otherwise looks good